pkg/apis/controller/common/v1beta1/common_types.go

```diff
@@ -220,8 +220,8 @@ const (
 	CustomCollector CollectorKind = "Custom"
 
 	// When model training source code persists metrics into persistent layer
-	// directly, metricsCollector isn't in need, and its kind is "noneCollector"
-	NoneCollector CollectorKind = "None"
+	// directly, sidecar container isn't in need, and its kind is "pushCollector"
+	PushCollector CollectorKind = "Push"
```

Comment:
Do you have any consensus on removing existing fields with @andreyvelich and @johnugeorge ?
In fact, removing the existing field means losing backward compatibility.
In general, the behavior is allowed us only when we upgrade API version (v1 -> v2).

Reply:
Yes, but I don't have strong concerns with removing None metrics collector.
I think, this parameter was never used by Katib users since it is not possible to run Katib Experiments when metrics are not collected.

Reply:
Since it is never used, I think, it is ok.